packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -45,6 +45,7 @@ const ARROW_KEY_MOVEMENT_LARGE = 5; // px
 const MAX_POINTER_TRAVEL_DISTANCE = 5; //px^2. determines how far the pointer can move to be treated as a drag vs. a click
 
 export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps> {
+    private _pasteOffset = 10;

```

Comment:
Is there any situation where we would want to change the paste offset? If not, it would be better to have it as a module const instead.

Reply:
Yes! The offset being applied is intentional. This is so that if a control is selected multiple times, it is easier to visually distinguish between the controls on screen. I did also send it to Patrick to confirm though, so if he wants to change the design I can do that